src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -93,7 +95,7 @@ public function __construct(string $name, string $value = null, $expire = 0, ?st
         }
 
         // from PHP source code
-        if (preg_match("/[=,; \t\r\n\013\014]/", $name)) {
+        if ($raw && preg_match('/['.self::$charNameReserved.']/', $name)) {
```

Comment:
let's replace the preg_match by `false !== strpbrk($name, self::RESERVED_CHARS)`